src/messaging/ExchangeContext.cpp

```diff
@@ -183,10 +183,17 @@ CHIP_ERROR ExchangeContext::SendMessage(Protocols::Id protocolId, uint8_t msgTyp
             return CHIP_ERROR_INTERNAL;
         }
 
+        //
+        // Our session should still be valid, since it had been evicted, all exchanges on that session would have been torn down
```

Comment:
```suggestion
        // Our session should still be valid, since if it had been evicted, all exchanges on that session would have been torn down
```